sklearn/inspection/_partial_dependence.py

```diff
@@ -954,3 +1111,470 @@ def plot(self, ax=None, n_cols=3, line_kw=None, contour_kw=None):
+                                            fixed_start_point=True,
+                                            feature_names=None, target=None,
+                                            n_cols=3, n_jobs=None, verbose=0,
+                                            line_kw=None, ax=None):
+    """Individual Conditional Expectation (ICE) plots.
+
+    The ``len(features)`` plots are arranged in a grid with ``n_cols``
+    columns. The deciles of the feature values will be shown with tick marks
+    on the x-axes.
+
+    .. note::
+
+        :func:`plot_individual_conditional_expectation` does not support using
+        the same axes with multiple calls. To plot the ICE curves for multiple
+        estimators, please pass the axes created by the first call to the
+        second call::
+
+          >>> from sklearn.inspection import (
+          ...                   plot_individual_conditional_expectation)
+          >>> from sklearn.datasets import make_friedman1
+          >>> from sklearn.linear_model import LinearRegression
+          >>> X, y = make_friedman1()
+          >>> est = LinearRegression().fit(X, y)
+          >>> disp1 = plot_individual_conditional_expectation(
+          ...     est, X)  # doctest: +SKIP
+          >>> disp2 = plot_individual_conditional_expectation(
+          ...     est, X,ax=disp1.axes_)  # doctest: +SKIP
+
+    Read more in the :ref:`User Guide <partial_dependence>`.
+
+    Parameters
+    ----------
+    estimator : BaseEstimator
+        A fitted estimator object implementing :term:`predict`,
+        :term:`predict_proba`, or :term:`decision_function`.
+        Multioutput-multiclass classifiers are not supported.
+
+    X : {array-like or dataframe} of shape (n_samples, n_features)
+        The data to use to build the grid of values on which the individual
+        conditional expectation will be evaluated.
+        This is usually the training data.
+
+    features : list of {int, str}
+        The target features for which to create the ICEs.
+        if any entry is a string, then it must be in ``feature_names``.
+
+    response_method : 'auto', 'predict_proba' or 'decision_function', \
+            optional (default='auto')
+        Specifies whether to use :term:`predict_proba` or
+        :term:`decision_function` as the target response. For regressors
+        this parameter is ignored and the response is always the output of
+        :term:`predict`. By default, :term:`predict_proba` is tried first
+        and we revert to :term:`decision_function` if it doesn't exist.
+
+    percentiles : tuple of float, optional (default=(0.05, 0.95))
+        The lower and upper percentile used to create the extreme values
+        for the ICE axes. Must be in [0, 1].
+
+    grid_resolution : int, optional (default=100)
+        The number of equally spaced points on the axes of the plots, for each
+        target feature.
+
+    fixed_start_point : bool, optional (default=True)
```

Comment:
to be removed

Reply:
Removed in https://github.com/scikit-learn/scikit-learn/pull/16164/commits/383ec6ceba008e06630d99c61820dfe721c58ba0